src/scripts/widget.js

```diff
@@ -0,0 +1,57 @@
+---
+---
+$(function() {
+    var WIDGET_DISPLAYED_FLAG = 'widget-displayed';
+
+    function sendGaEvent(event) {
+        {% if jekyll.environment == "production" %}
+
+        ga('send', 'event', 'widget', event);
+
+        {% endif %}
+    }
+
+    function updateWidgetVisibility() {
+        if(window.needShowShareWidget()) {

```

Comment:
If you already shown the widget there is no need to check `needShowShareWidget`. Just show it again